compose/src/main/java/com/bottlerocketstudios/compose/yelp/YelpComponents.kt

```diff
@@ -0,0 +1,85 @@
+package com.bottlerocketstudios.compose.yelp
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.bottlerocketstudios.compose.R
+import com.bottlerocketstudios.compose.map.YelpCardLayout
+import com.bottlerocketstudios.compose.resources.Dimens
+import com.bottlerocketstudios.mapsdemo.domain.models.Business
+import com.bottlerocketstudios.mapsdemo.domain.models.YelpMarker
+import kotlinx.coroutines.launch
+import timber.log.Timber
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun ColumnScope.YelpBusinessList(businessList: List<Business>, selectedYelpMarker: YelpMarker, onClick: (Business) -> Unit) {
+
+    val noPosition = -1
+    val state = rememberLazyListState()
+    val coroutineScope = rememberCoroutineScope()
+    val itemPosition = remember {
+        mutableStateOf(value = -1)
+    }
+
```

Comment:
Add comment here explaining that this how you implemented the scroll to selected item.